jitpack.yml

```diff
@@ -0,0 +1,8 @@
+jdk:
+  - openjdk8
+install:
+  - echo "Building non-compose artifacts"
+  - ./gradlew clean install -xtest
+  - cp -R libraries libraries-non-compose
```

Comment:
By copying this to a different directory, we avoid deleting it when we do a gradle clean, but thankfully Jitpack still finds it!